server/server.go

```diff
@@ -166,10 +166,14 @@ type lazyHandler struct {
 	engine  *negroni.Negroni
 }
 
+var once sync.Once
+
 func (lazy *lazyHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	for _, f := range lazy.options {
-		f()
-	}
+	once.Do(func() {
```

Comment:
If multiple server instances are started during the test, they will only be executed once.

Reply:
cc @rleungx